tests/TestCase/View/Helper/BreadcrumbsHelperTest.php

```diff
@@ -22,9 +23,9 @@ public function setUp()
 
         $this->View = new View();
         if (class_exists('\Cake\View\Helper\BreadcrumbsHelper')) {
```

Comment:
The `develop` branch is for Cake `3.5+` which has the `BreadcrumbsHelper` hence this check can be removed.